workflow-fingerprint-collector/src/main/java/ca/on/oicr/pde/workflows/FingerprintCollectorWorkflow.java

```diff
@@ -462,4 +458,5 @@ private String makeBasename(final String name) {
         }
         return basename;
     }
+    //FIXME
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Take the required action to fix the issue indicated by this comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1134)